### PR TITLE
{bio}[intel/2019a] BRAKER v2.1.2, GeneMark-ET v4.38, GenomeThreader v1.7.1, spaln 2.3.3c

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
~#8417~ (Bioptyhon), ~#8429~ (SAMtools), ~#8431~ (BamTools), ~#8436~ (AUGUSTUS), ~#8434~ (Exonerate), ~#8435~ (BLAST+), ~~#8432~~ (`Scalar::Util::Numeric` in `Perl`)

### CODE REVIEW

File: easybuild/easyconfigs/b/BRAKER/BRAKER-2.1.2-intel-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+sources = ['v%(version)s.tar.gz']
+checksums = ['9f178c5fe64ae358dcba9936802d24e330312e698a3f7930d1f91e58974129d3']
+
+dependencies = [
+    ('Perl', '5.28.1'),
+    ('AUGUSTUS', '3.3.2'),
+    ('GeneMark-ET', '4.38'),
+    ('BamTools', '2.5.1'),
+    ('SAMtools', '1.9'),
+    ('GenomeThreader', '1.7.1', '', True),
```
Comment (vanzod) [inline easybuild/easyconfigs/b/BRAKER/BRAKER-2.1.2-intel-2019a.eb]: The version suffix `-Linux_x86_64-64bit` is missing here for `GenomeThreader`

Comment (boegel) [reply easybuild/easyconfigs/b/BRAKER/BRAKER-2.1.2-intel-2019a.eb]: fixed in d826c0c9bf

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2009.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3321f9eb87ff01303e5cbecc53a1af7f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (4 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/5bbc353cd2af6088a5aee725fc86d9d2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 14 out of 16 (4 easyconfigs in this PR)
node2408.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/5fc7c68c3f030047a562ef6f429b2beb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (4 easyconfigs in this PR)
node2434.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d4b1e8e6ae471572acbcde4a54e4439c for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 5 out of 7 (4 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/39215299b0d605512005c9b6f893880d for a full test report.

**Comment by vanzod** [comment]
@boegel `Exonerate` requires `pkg-config` as build dependency.

**Comment by boegel** [comment]
@vanzod Ugh, ok, I'll get that fixed in a separate PR...

**Comment by boegel** [comment]
@vanzod `pkg-config` build dep for `Exonerate` added in #8776

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/9cfbfa78a54a202ceb173184c7b66be2 for a full test report.

**Pull request merged**